config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -430,6 +430,209 @@
       "timeFrom": null,
       "timeShift": null
     },
+    {
+      "id": 31,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 17
+      },
+      "type": "graph",
+      "title": "#2b Time to Start PipelineRun With Ephemeral Env",
+      "thresholds": [
+        {
+          "$$hashKey": "object:1475",
+          "colorMode": "critical",
+          "fill": true,
+          "line": true,
+          "op": "gt",
+          "value": 5,
+          "yaxis": "left"
+        }
+      ],
+      "pluginVersion": "9.1.6",
+      "description": "Measure the time between ApplicationSnapshot creation to pipelineRun creation when the pipelineRun has an ephemeral environment. \n",
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "aliasColors": {},
+      "dashLength": 10,
+      "fill": 1,
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "pointradius": 2,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "histogram_quantile(0.90, sum(rate(snapshot_created_to_pipelinerun_with_ephemeral_env_started_seconds_bucket[$__rate_interval])) by (le))",
+          "interval": "",
+          "legendFormat": "percentile90",
+          "refId": "A",
+          "datasource": {
```

Comment:
Datasource must be removed from JSON
https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring#4-grafana-dashboards